docs/local_install_skaffold.md

```diff
@@ -1,51 +1,280 @@
-# Local install of Substra using kubernetes and skaffold
+# Local install of Substra using Kubernetes and Skaffold
+
+> This is an ongoing document, please feel free to reach us or to raise any issue.
+
+This guide will help you to run the Substra platform on your machine in development mode, with a two organisations setup.
+
+## Index
+- [Local install of Substra using Kubernetes and Skaffold](#local-install-of-substra-using-kubernetes-and-skaffold)
+  - [Index](#index)
+  - [1. Requirements](#1-requirements)
+    - [General knowledge](#general-knowledge)
+    - [Hardware requirements](#hardware-requirements)
+    - [Software requirements](#software-requirements)
+      - [1. Kubernetes](#1-kubernetes)
+        - [Mac](#mac)
+        - [Ubuntu](#ubuntu)
+      - [2. Minikube](#2-minikube)
+      - [3. Helm](#3-helm)
+      - [4. Skaffold](#4-skaffold)
+    - [Virtualisation](#virtualisation)
+  - [2. Setup](#2-setup)
+    - [1. Get the source code](#1-get-the-source-code)
+    - [2. Configuration](#2-configuration)
+      - [Minikube](#minikube)
+      - [Network](#network)
+        - [Ubuntu](#ubuntu-1)
+        - [Mac](#mac-1)
+  - [3. Running the platform](#3-running-the-platform)
+    - [Start Substra](#start-substra)
+    - [Stop Substra](#stop-substra)
+  - [4. Login, password and urls](#4-login-password-and-urls)
+    - [Credentials and urls](#credentials-and-urls)
+    - [Browser extension](#browser-extension)
+    - [Substra CLI config & login](#substra-cli-config--login)
+  - [5. Titanic example](#5-titanic-example)
+  - [6. Troubleshooting](#6-troubleshooting)
+    - [Virtualisation](#virtualisation-1)
+    - [Kubectl useful commands](#kubectl-useful-commands)
+    - [Minikube useful commands](#minikube-useful-commands)
+    - [Tiller](#tiller)
+    - [Ongoing issues](#ongoing-issues)
+  - [7. Further resources](#7-further-resources)
+  - [8. Need help](#8-need-help)
+  - [9. Acknowledgements](#9-acknowledgements)
+___
+
+When everything is ready, you will be able to start Substra with:
+```sh
+minikube start --cpus 6 --memory 8192 --disk-size 50g --kubernetes-version='v1.15.4'
+
+skaffold dev # in hlf-k8s, susbtra-backend, substra-frontend repositories
```

Comment:
You could make this even more explicit by telling users to run `skaffold dev` in all 3 directories in independent shells otherwise they may wait for the first one to finish to launch the next one.

Reply:
You're right!